test/show.jl

```diff
@@ -221,7 +221,7 @@ end
         @test sprint(show, df, context=:color=>true) == """
             \e[1m2×2 DataFrame\e[0m
             \e[1m Row \e[0m│\e[1m Fish   \e[0m\e[1m Mass      \e[0m
-            \e[1m     \e[0m│\e[90m String \e[0m\e[90m Float64?  \e[0m
+                 │\e[90m String \e[0m\e[90m Float64?  \e[0m
```

Comment:
what are the reasons for these changes?

Reply:
It was a bug. We were applying the same header highlighting to an empty cell.